stratego/lib/stratego/squares/supervisor.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Stratego.Squares.Supervisor do
+    use Supervisor
+
+    def start_link() do
+        Supervisor.start_link( __MODULE__, :no_args, name: __MODULE__ )
+    end
+
+    def init( :no_args ) do
+        children = 
+            for x <- squares() do
+              worker( Stratego.Squares, [x], id: x )
+            end
+
```

Comment:
I don't think one_for_one is the correct strategy,

If one square dies, doesn't that mean the game is now invalid, and all squares should die, too?